### DISCUSSION TITLE
Linq2DB still support SAP HANA ?

### DISCUSSION THREAD

**Discussion by rahuljain257** [linq2db/linq2db#4270] [General]
Hello Team,

I hope everyone are doing well !!! I was going through the documentation - https://linq2db.github.io/ and **HANA** is not mentioned anywhere But the old version used to support.

So my query are as follows - 

1. HANA is still supported ?
2. Add / Update / Delete operation is supported for HANA ?

Regards
Rahul Jain

**Comment by sdanyliv** [comment]
Yes it is still supported:
https://github.com/linq2db/linq2db/tree/master/Source/LinqToDB/DataProvider/SapHana

**Comment by MaceWindu** [comment]
Yep, SAP HANA supported and we test linq2db against it. Both native ADO.NET provider and ODBC providers supported.

Talking about limitations I could remember only one major issue - lack of support for array types as SAP provider doesn't implement them and it should be implemented on linq2db side, which is possible but doesn't look like high priority thing to us.
